controller/konnect/ops/ops.go

```diff
@@ -216,7 +250,13 @@ func shouldUpdate[
 func Update[
 	T constraints.SupportedKonnectEntityType,
 	TEnt constraints.EntityType[T],
-](ctx context.Context, sdk SDKWrapper, syncPeriod time.Duration, cl client.Client, e *T) (ctrl.Result, error) {
+](
+	ctx context.Context,
+	sdk SDKWrapper,
+	syncPeriod time.Duration,
+	cl client.Client,
+	e TEnt,
+) (ctrl.Result, error) {
 	var (
 		ent = TEnt(e)
```

Comment:
```suggestion
```
Since we changed the function signature, the `ent` is not required and the appearance of `ent` should be changed to `e`.